app/controllers/orders/client_summary.js

```diff
@@ -6,6 +6,22 @@ const { getOwner } = Ember;
 export default detail.extend({
   showBeneficiaryModal: false,
 
+  titles: Ember.computed(function(){
+    return [
+      { name: "Mr", id: "Mr" },
+      { name: "Mrs", id: "Mrs" },
+      { name: "Miss", id: "Miss" },
+      { name: "Ms", id: "Ms" }
+    ];
+  }),
+
+  identityTypes: Ember.computed(function() {
+    return [
+      { name: 'Hong Kong Identity Card', id: 1 },
```

Comment:
I believe for identity type we have records in db. I am not sure we are preloading those or not in ember-data but I feel instead of hardcoded values picking those from ember-data will be better. Let me know your thoughts. :)

Reply:
We only load `booking_type` in preload and I think hardcoded value is okay for select list.

Reply:
Can we preload identity-type? What if someone deletes and recreates records then Id's will change I think. So better to send ids of the actual record. Any thoughts?

We previously did same with booking_type then did it as per actual records.

If you still think it's fine then keep it. 👍

Reply:
I second @namrataukirde on this.